osu.Game.Rulesets.Osu/UI/IHitPolicy.cs

```diff
@@ -20,7 +20,7 @@ public interface IHitPolicy
         /// <param name="hitObject">The <see cref="DrawableHitObject"/> to check.</param>
         /// <param name="time">The time to check.</param>
         /// <returns>Whether <paramref name="hitObject"/> can be hit at the given <paramref name="time"/>.</returns>
-        bool IsHittable(DrawableHitObject hitObject, double time);
+        ClickAction CheckHittable(DrawableHitObject hitObject, double time);
```

Comment:
Open to renamings of both `CheckHittable` and `ClickAction`. `ClickAction` was brought across from the stable naming.